lib/ss_light.go

```diff
@@ -383,18 +393,24 @@ func (l *LightClient) Start(
 
 	_, err = io.Copy(dst, rsc)
 	if err != nil {
-		return "Failed writing to destination", err
+		return NewOut(internalError, "Failed writing to destination", err.Error(), nil)
 	}
+	downloadTime := time.Now().Unix() - startTime
 
+	// STEP : Waiting for micropayments clean up
+	showStep(success, "Finishing download", l.jsonOut)
 	// Wait 5 secs for SCP to send all MPs. This can be optimized
 	<-time.After(time.Second * 5)
 
-	err = updateInfo(metadata)
+	err = updateInfo(metadata, downloadTime)
 	if err != nil {
 		log.Warn("Failed updating metadata after download Err: %s", err.Error())
 	}
+	// STEP : Updated Cookie
+	showStep(success, "Updating cookie", l.jsonOut)
```

Comment:
Remove this. This is sent after Cookie is updated. Cookie doesn't mean anything to the user of this.